cmd/push.go

```diff
@@ -298,6 +298,12 @@ func (p *PushCmdParams) PostInteractive(_ ActionCtx) error {
 }
 
 func (p *PushCmdParams) Validate(ctx ActionCtx) error {
+	if p.timeout < 1 {
```

Comment:
Don't you think it would be better to simply return error here to not confuse users? Especially for timeouts > 60s.

Reply:
I think clamping for this is not a problem - simply doing a check in case they gave a negative value etc.